gatekeeper/Dockerfile

```diff
@@ -10,16 +10,20 @@ LABEL Name=keycloak-gatekeeper \
       Url=https://github.com/keycloak/keycloak-gatekeeper \
       Help=https://issues.jboss.org/projects/KEYCLOAK
 
-RUN adduser -D -u 1000 gatekeeper
+USER root
 
-RUN apk add --no-cache ca-certificates curl tar
+WORKDIR "/opt/jboss/gatekeeper"
 
-WORKDIR "/opt"
+RUN echo "jboss:x:1000:jboss" >> /etc/group
```

Comment:
User shouldn't be jboss, but should be gatekeeper.

Reply:
changed